Resources/Prototypes/GameRules/events.yml

```diff
@@ -69,6 +69,30 @@
     earliestStart: 15
     minimumPlayers: 15
 
+- type: gameRule
+  id: SpaceNinjaSpawn
+  config:
+    !type:NinjaRuleConfiguration
+    id: SpaceNinjaSpawn
+    weight: 10
+    endAfter: 1
+    earliestStart: 60
+    minimumPlayers: 15
+    objectives:
+    - DownloadObjective
+    - DoorjackObjective
+    - SpiderChargeObjective
+    - TerrorObjective
+    - SurviveObjective
+    implants: [ MicroBombImplant ]
+    threats:
+    - announcement: terror-dragon
+      rule: Dragon
+    - announcement: terror-revenant
+      rule: RevenantSpawn
+    - announcement: terror-pirates
+      rule: Pirates
```

Comment:
this system of threats seems exceptionally strong. being able to start midround antags is a powerful tool for anyone.

Reply:
what should i have instead of these (though i should pirates since it would be too strong)